wagtailvideos/jinja2tags.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import, unicode_literals
+
+import mimetypes
+
+from jinja2.ext import Extension
+
+from django import template
+from django.forms.widgets import flatatt
+from django.utils.text import mark_safe
+
+
+def video(video, attrs):
+    if not video:
```

Comment:
We probably want to error here. An empty string is not very helpful when debugging. The original didn't do this but a type check would also be handy